app/database/queries/breeze_buddy/merchant_entities.py

```
@@ -0,0 +1,342 @@
+        SELECT id, merchant_id, name, description, shop_identifiers,
+               is_active, owner_id, created_at, updated_at
+        FROM merchants
+        WHERE merchant_id = ANY($1)
+        ORDER BY name ASC
+    """
+
+    try:
+        async for conn in get_db_connection():
+            rows = await conn.fetch(query, merchant_ids)
+            return [_build_merchant_entity_response(row) for row in rows]
+        return []  # Fallback if connection not available
```

Ruff (0.14.14)

[warning] Consider moving this statement to an `else` block

(TRY300)